rwengine/src/script/ScriptFunctions.hpp

```diff
@@ -139,32 +139,23 @@ inline GameString gxt(const ScriptArguments& args, const ScriptString id) {
     return args.getWorld()->data->texts.text(id);
 }
 
-inline BlipData& createObjectBlip(const ScriptArguments& args,
-                                  GameObject* object) {
+inline BlipData& createBlip(const ScriptArguments& args, const ScriptVec3& coord,
+                            BlipData::BlipType type) {
     BlipData data;
-    switch (object->type()) {
-        case GameObject::Vehicle:
-            data.type = BlipData::Vehicle;
-            data.colour = 0;  // @todo 4 in Vice City
-            break;
-        case GameObject::Character:
-            data.type = BlipData::Character;
-            data.colour = 1;  // @todo 4 in Vice City
+    data.coord = coord;
+    data.type = type;
+    switch (type) {
+        case BlipData::Contact:
+            data.colour = 2;
             break;
-        case GameObject::Pickup:
-            data.type = BlipData::Pickup;
-            data.colour = 6;  // @todo 4 in Vice City
-            break;
-        case GameObject::Instance:
-            data.type = BlipData::Instance;
-            data.colour = 6;  // @todo 4 in Vice City
+        case BlipData::Coord:
+            data.colour = 5;
```

Comment:
nit: The Vice City comments were usefull in my opinion.

Reply:
Ugh, it was a long time ago, so I don't really remember what was the purpose of this change. BUT I think there was missing bilp type. Although, these values are from original game.

Reply:
It seems a function was added and the "deleted" values have been revised and git has, instead of being easy-to-read chosen to be efficient and reuse as many lines as possible. Looks a mess, but if one reads more code it stands clear that nothing was really lost, just re-organized. :)